### DISCUSSION TITLE
WS2812B LED Strip reading data but not turning on

### DISCUSSION THREAD

**Discussion by phamkev** [awawa-dev/HyperHDR#597] [General]
Currently using a Raspberry Pi 4b and a WS2812B LED Strip connected to an AC/DC 5V10A power supply (BTF-LIGHTNING brand from Amazon). When connected, the power supply can turn on the Raspberry Pi but not the led strip. I've tried multiple different female barrel adapters and have replaced the LED strip twice without success. I have a jumper wire connecting pin 09 (ground) to the same barrel connector ground and have a jumper wire from pin 13 (GIPO 27) connected to the data port in the LEDs. 

Here are the screenshots on Hyperion. https://imgur.com/a/FOHChaX

Not sure how to adjust the voltage limiter on hyperion.

**Comment by awawa-dev** [comment]
Hi
As you can see in the attached screenshot (or read in the FAQ in the wiki https://github.com/awawa-dev/HyperHDR/wiki/FAQ about it and required root privileges) the method you use is not officially supported in **HyperHDR**. Why? Because it causes many problems and if something goes wrong it's very diffucult for standard user to diagnose them. Read about HyperSerialPico as an alternative LED driver: https://github.com/awawa-dev/HyperHDR/discussions/561. BTW your WS2812B needs 5V data line and Rpi GPIO offers only 3.3V so you need a 3.3V to 5V level shifter anyway (sometimes it may work without it but it's a matter of luck). Or use HyperSerialPico and recommended Raspberry Pi Pico board with a built-in 3.3V to 5V level shifter.
